.github/workflows/auto-cc.yml

```diff
@@ -10,23 +10,27 @@ jobs:
   auto-notify:
     if: |
         github.repository_owner == 'cupy' &&
-        github.event.label.name == 'hip'
+        contains('|hip|array-api|', github.event.label.name)
+    env:
+      NOTIFY_TARGETS: '{"hip": "@amathews-amd", "array-api": "@asmeurer"}'
```

Comment:
Would this work? 😛
```suggestion
      NOTIFY_TARGETS: '{"hip": "@amathews-amd", "array-api": "@asmeurer @leofang"}'
```

Reply:
Sure! I thought you are watching everything 😉